ansible/library/killbill_facts

```diff
@@ -27,11 +16,11 @@ end
 require 'kpm'
 require 'kpm/version'
 
-kpm_facts = with_captured_stdout { KPM::System.new.information(data['bundles_dir'],
+kpm_facts = KPM::System.new.information(data['bundles_dir'],
```

Comment:
Shouldn't `'kpm' => kpm_facts` below be changed as well? `kpm_facts` is already json, so when calling `JSON.dump`, the `kpm` value will be a string, not a JSON object.